src/model/comment.ml

```diff
@@ -51,7 +51,10 @@ type tag =
   [ `Author of string
   | `Deprecated of nestable_block_element with_location list
   | `Param of string * nestable_block_element with_location list
-  | `Raise of string * nestable_block_element with_location list
+  | `Raise of
+    [ `Code_span of string | `Reference of Reference.t * link_content ]
```

Comment:
Cosmetic changes: You can create a new type that is included here and in `inline_element` to avoid duplication:
```
type reference_element = [ `Reference of Reference.t * link_content ]
```

Either that or a type that doesn't have an empty content.

Reply:
Done!